README.md

```diff
@@ -24,18 +24,31 @@ You can use your own computer as a server, but for other people to connect to yo
   [5]: http://en.wikipedia.org/wiki/Port_forwarding
 
 
-Installing
+Previous Requisites
 ------------------------------------------------------------------------
+Supported Node version is latest LTS 6.10.3. You can set the specific node version using:
+- [nvm](https://github.com/creationix/nvm) for Linux or Mac OS
+- [nvm-windows](https://github.com/coreybutler/nvm-windows) for Windows users
 
-    ./pokemon-showdown
+Then you can install this project's node version by running:
+```
+nvm install
+```
```

Comment:
Also, there is a big difference between our earliest supported Node version (v6.0.0) and our recommended Node version (latest release, currently v7.10.0).